demos/python_demos/rag_chatbot/servable_stream/model.py

```diff
@@ -204,49 +185,140 @@ def serialize_completions(batch_size, result):
     return [Tensor("completion", serialize_byte_tensor(
         np.array(result, dtype=np.object_)).item())]
 
-
+def download_documents(file, target_folder):
+    if not os.path.exists(target_folder):
+        os.makedirs(target_folder)
+    print("file", file, "target_folder", target_folder)
```

Comment:
Is it needed? If not, remove, if needed, it will not appear without flush=True @mzegla